contracts/TypedMemView.sol

```diff
@@ -360,7 +352,7 @@ library TypedMemView {
      * @return          bool - True if the 5-byte type flag is equal
      */
     function sameType(bytes29 left, bytes29 right) internal pure returns (bool) {
-        return (left ^ right) >> (2 * TWELVE_BYTES) == 0;
+        return (left ^ right) >> SHIFT_TO_TYPE == 0;
```

Comment:
wait, i don't understand - why are you shifting 216 instead of 27

Reply:
`TWELVE_BYTES` is 96 bits. Previous shift is 2*96. Correct shift is 2*96 + 24